packages/web-twig/src/Resources/components/Toast/stories/ToastAlignment.twig

```diff
@@ -0,0 +1,85 @@
+<!-- Toast alignment demo: start -->
+<script>
+    const setToastAlignmentX = (event, selector) => {
+        const toastElement = document.querySelector(selector);
+
+        toastElement.classList.remove('Toast--left', 'Toast--center', 'Toast--right');
+        toastElement.classList.add(`Toast--${event.target.value}`);
+    };
+
+    const setToastAlignmentY = (event, selector) => {
+        const toastElement = document.querySelector(selector);
+
+        toastElement.classList.remove('Toast--top', 'Toast--bottom');
+        toastElement.classList.add(`Toast--${event.target.value}`);
+    };
+</script>
+
+<form>
+    <fieldset onchange="setToastAlignmentY(event, '#toast-example')" style="border: 0;">
+        <legend>Vertical alignment:</legend>
+        <Radio name="toast-alignment-y" autocomplete="off" UNSAFE_className="mr-600" value="top" id="toast-alignment-top" label="Top" />
```

Comment:
please use styling props instead of unsafe classname

```suggestion
        <Radio name="toast-alignment-y" autocomplete="off" marginRight="600" value="top" id="toast-alignment-top" label="Top" />
```

Reply:
Everywhere :)